storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1518,6 +1520,12 @@ static MYSQL_SYSVAR_BOOL(error_on_suboptimal_collation,
                          "collation is used",
                          nullptr, nullptr, false);
 
+static MYSQL_SYSVAR_BOOL(
+    no_create_column_family, rocksdb_no_create_column_family,
+    PLUGIN_VAR_OPCMDARG | PLUGIN_VAR_READONLY,
+    "Do not allow creation of new Column Families from client interface.", nullptr,
```

Comment:
"through index comments"? Are there other affected client interfaces?

Reply:
Currently, no.  This CREATE|ALTER index comment is the only path through here where the create may be prohibited.  I considered a more specific message but thought that the more generic message may be more appropriate since we may (probably will) create or invent new ways for explicit Column Family creation and management in the future and will probably forget to re-word the message.

Re-reading this comment I realize that's a lot of  'probably' and 'ifs'.  So yeah, maybe your right.  A specific message for now and worry about rewording later on if/when we do revisit CF management.